docs/api/datasets/non_geo_datasets.csv

```diff
@@ -15,6 +15,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `DFC2022`_,S,Aerial,"CC-BY-4.0","3,981",15,"2,000x2,000",0.5,RGB
 `Digital Typhoon`_,"C, R",Himawari,"CC-BY-4.0","189,364",8,512,5000,Infrared
 `DL4GAM`_,S,"Sentinel-2","CC-BY-4.0","2,251 or 11,440","2","256x256","10","MSI"
+`DOTA`_,OD,"Google Earth, Gaofen-2, Jilin-1, CycloMedia B.V.","CC-BY-NC-4.0","5,229",15,"1000-5000",RGB
```

Comment:
I think we should change the license to "non-commercial" since no specific license was defined. Or leave it blank until a single specific license is chosen.

Reply:
```suggestion
`DOTA`_,OD,"Google Earth, Gaofen-2, Jilin-1, CycloMedia B.V.",non-commercial,"5,229",15,1000--5000,RGB
```